ground/src/main/java/com/google/android/ground/model/locationofinterest/LocationOfInterest.kt

```diff
@@ -51,7 +49,9 @@ data class LocationOfInterest(
    * Whether this LOI was created opportunistically by the user through the Suggest LOI flow, or
    * false if the LOI was created by the survey organizer.
    */
-  val isOpportunistic: Boolean = false
+  val isOpportunistic: Boolean = false,
+  /** Custom map of properties for this LOI. Corresponds to the properties field in GeoJSON */
+  val properties: Map<String, Any>? = null,
```

Comment:
Can we define this type as a typealias? Perhaps `LocationOfInterest.Properties`?

Reply:
Done